annotations/static/annotations/css/annotators/text.css

```diff
@@ -25,6 +25,9 @@
 
 .relation-list {
     font-size: 0.8em;
+    height: 90vh;
```

Comment:
Ideally this should take up all the space there is, not a defined height. also there is still an issue with the footer:
![image](https://user-images.githubusercontent.com/8881141/32400911-b878ada4-c0c3-11e7-825e-3a3b0c87cd59.png)

Reply:
I am having trouble figuring out what is wrong with this fix. I tried it on the display size for a 13in screen, ipad and ipad pro screens and all work as intended. Also vh is a percentage of the users viewport. So this element should take up 90% percent of the screen. If you want it to take up 100% of the screen.The affix property of the list/search element will have to be modified to "turn off" when a certain point is reached so that the whole thing scrolls up a little when the footer comes up otherwise the element will always stay fixed and be in the way. 


![screen shot 2018-01-11 at 10 26 30 am](https://user-images.githubusercontent.com/16781171/34838253-a75ce9fe-f6ba-11e7-996b-a12f873a7c6b.png)
# Ipad
![screen shot 2018-01-11 at 10 20 20 am](https://user-images.githubusercontent.com/16781171/34838272-bb2b345e-f6ba-11e7-9e27-79b52e9d6a8b.png)

# Ipad Pro
![screen shot 2018-01-11 at 10 19 46 am](https://user-images.githubusercontent.com/16781171/34838314-e501d102-f6ba-11e7-856a-afb409e72dd8.png)

# 13in Screen
![screen shot 2018-01-11 at 10 19 09 am](https://user-images.githubusercontent.com/16781171/34838343-f2a941fa-f6ba-11e7-8fa3-0fa4d399202a.png)

Reply:
hm, ja, not sure how I got the problem. Maybe I confused app versions. Let' s leave it the way it is. If it does turn out to have issues, we can always come back to it. Please reopen the PR.